.github/workflows/python-module.yml

```diff
@@ -0,0 +1,147 @@
+    - name: Check formatting
+      run: |
+        docker run \
+          -v$(pwd):/app \
+          -w/app/python \
+          testing \
+          bash -c "\$PYBIN/black --check ."
+    - name: Try to import on different python versions
+      run: |
+        docker run \
+          -v$(pwd):/app \
+          testing \
+          bash -c "
+            for PYBIN in /opt/python/{cp34-cp34m,cp35-cp35m,cp36-cp36m,cp37-cp37m}/bin; do
+              \$PYBIN/pip install /app/python/dist/wkw-*-py3-none-manylinux1_x86_64.whl
+              \$PYBIN/python -c 'import wkw; print(wkw.Dataset)'
+            done"
+    - name: Publish
+      if: startsWith(github.event.ref, 'refs/tags')
+      run: |
+         echo "[distutils]" >> .pypirc
+         echo "index-servers = pypi" >> .pypirc
+         echo "[pypi]" >> .pypirc
+         echo "repository=https://upload.pypi.org/legacy/" >> .pypirc
+         echo "username=${{ secrets.PYPI_USERNAME }}" >> .pypirc
+         echo "password=${{ secrets.PYPI_PASSWORD }}" >> .pypirc
+   
+         docker run \
+           -v$(pwd):/app \
+           -v$(pwd)/.pypirc:/root/.pypirc \
+           -w/app/python \
+           testing \
+           bash -c "\$PYBIN/python setup.py bdist_wheel -p manylinux1_x86_64 upload -r pypi"
+
+  build_mac:
+    runs-on: macOS-10.14
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x' # Version range or exact version of a Python version to use, using semvers version range syntax.
+        architecture: 'x64' # (x64 or x86)
+    - name: Install Rust and lz4
+      run: |
+        brew install rustup-init lz4
+        rustup-init -y
+    - name: Build
+      run: |
+        cd python
+        export PATH=$PATH:$HOME/.cargo/bin
+        python setup.py install
+    - name: Test
+      run: |
+        cd python
+        python -c "import wkw"
+        pip install pytest
+        pytest tests
+    - name: Publish
+      if: startsWith(github.event.ref, 'refs/tags')
+      run: |
+        echo "[distutils]" >> $HOME/.pypirc
+        echo "index-servers = pypi" >> $HOME/.pypirc
+        echo "[pypi]" >> $HOME/.pypirc
+        echo "repository=https://upload.pypi.org/legacy/" >> $HOME/.pypirc
+        echo "username=${{ secrets.PYPI_USERNAME }}" >> $HOME/.pypirc
+        echo "password=${{ secrets.PYPI_PASSWORD }}" >> $HOME/.pypirc
+
+        cd python
+        export PATH=$PATH:$HOME/.cargo/bin
+        pip install wheel
+        python setup.py bdist_wheel -p $(python -c "import distutils.util; print(distutils.util.get_platform())") upload
+
+  build_win:
+    runs-on: windows-2019
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x' # Version range or exact version of a Python version to use, using semvers version range syntax.
+        architecture: 'x64' # (x64 or x86)
+    - name: Install lz4
+      shell: bash
```

Comment:
Is this the linux subsystem bash? Will wkw work in the Windows command line, too? Or isn't this a requirement?

Reply:
I don't think it is the linux subsystem, but just a bash

Reply:
I thought, `bash` on windows always refers to the linux-based one. Also commands like `mv` don't exist in the windows cmd as far as I know.